src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -46,6 +48,52 @@
     "flags": ([str], "List of active flags")
 }
 
+device_status_template = {
+    'options': {
+        'cost-capacity': {
+            'name': 'cost-capacity',
+            'available': False,
+            'capacity': 0,
+            'used': 0,
+            'hdd_used': 0,
+            'ssd_used': 0,
+            'nvme_used': 0,
+        },
+        'throughput': {
+            'name': 'throughput-optimized',
+            'available': False,
+            'capacity': 0,
+            'used': 0,
+            'hdd_used': 0,
+            'ssd_used': 0,
+            'nvme_used': 0,
+        },
+        'iops': {
+            'name': 'iops-optimized',
+            'available': False,
+            'capacity': 0,
+            'used': 0,
+            'hdd_used': 0,
+            'ssd_used': 0,
+            'nvme_used': 0,
+        },
+    },
+    'recommended_option': 0
```

Comment:
when I tested the output of the recommended option was a string. So shouldn't this be null or None or something like that?
```suggestion
    'recommended_option': None
```